f5-bigip-mixin/dashboards/f5-virtual-server-overview.libsonnet

```diff
@@ -0,0 +1,606 @@
+  datasource: promDatasource,
+  targets: [
+    prometheus.target(
+      'bigip_vs_status_availability_state{job=~"$job", instance=~"$instance", vs=~"$f5_virtual_server"}',
+      datasource=promDatasource,
+      legendFormat='{{vs}} - {{partition}} - {{instance}}',
+      format='time_series',
+    ),
+  ],
+  type: 'stat',
+  title: 'Availability status',
+  description: 'The availability state of the virtual server.',
+  fieldConfig: {
+    defaults: {
+      color: {
+        mode: 'thresholds',
+      },
+      mappings: [
+        {
+          options: {
+            '0': {
+              color: 'red',
+              index: 1,
+              text: 'Unavailable',
+            },
+            '1': {
+              color: 'green',
+              index: 0,
+              text: 'Available',
+            },
+          },
+          type: 'value',
+        },
+      ],
+      thresholds: {
+        mode: 'absolute',
+        steps: [
+          {
+            color: 'green',
+            value: null,
+          },
+        ],
+      },
+      unit: '',
+    },
+    overrides: [],
+  },
+  options: {
+    colorMode: 'value',
+    graphMode: 'none',
+    justifyMode: 'auto',
+    orientation: 'auto',
+    reduceOptions: {
+      calcs: [
+        'lastNotNull',
+      ],
+      fields: '',
+      values: false,
+    },
+    textMode: 'auto',
+  },
+  pluginVersion: '10.2.0-60139',
+};
+
+local requests__intervalPanel = {
+  datasource: promDatasource,
+  targets: [
+    prometheus.target(
+      'increase(bigip_vs_tot_requests{job=~"$job", instance=~"$instance", vs=~"$f5_virtual_server"}[$__interval:])',
+      datasource=promDatasource,
+      legendFormat='{{vs}} - {{partition}} - {{instance}}',
+      format='time_series',
+      interval='1m',
+    ),
+  ],
+  type: 'timeseries',
+  title: 'Requests / $__interval',
+  description: 'The number of requests made to the virtual server.',
+  fieldConfig: {
+    defaults: {
+      color: {
+        mode: 'palette-classic',
+      },
+      custom: {
+        axisCenteredZero: false,
+        axisColorMode: 'text',
+        axisLabel: '',
+        axisPlacement: 'auto',
+        barAlignment: 0,
+        drawStyle: 'line',
+        fillOpacity: 30,
+        gradientMode: 'none',
+        hideFrom: {
+          legend: false,
+          tooltip: false,
+          viz: false,
+        },
+        insertNulls: false,
+        lineInterpolation: 'smooth',
+        lineWidth: 2,
+        pointSize: 5,
+        scaleDistribution: {
+          type: 'linear',
+        },
+        showPoints: 'never',
+        spanNulls: false,
+        stacking: {
+          group: 'A',
+          mode: 'none',
+        },
+        thresholdsStyle: {
+          mode: 'off',
+        },
+      },
+      mappings: [],
+      thresholds: {
+        mode: 'absolute',
+        steps: [
+          {
+            color: 'green',
+            value: null,
+          },
+        ],
+      },
+      unit: 'reqps',
```

Comment:
think this unit should be changed unless you want to make this a rate panel, which i think you easily could, your call

Reply:
updated to none instead of requests.